app/enums/codes.ts

```diff
@@ -176,6 +176,12 @@ export enum RequestTypeCode {
   BERE = 'BERE',
   /** BC Benefit Company - BC to Benefit Conversion */
   BECV = 'BECV',
+  /** BC Benefit Company - Community Contribution Company Conversion */
+  BECC = 'BECC',
+  /** Unlimited Liability Company - BC Limited Conversion */
+  ULCB = 'ULCB',
+  /** Unlimited Liability Company - Benefit Company Conversion */
+  ULBE = 'ULBE',
```

Comment:
Does order matter?

I was thinking that these should be grouped together -- Benefit Company codes together, ULC codes together. Thoughts?

Reply:
It might look nicer, but I don't know if it would make things easier to find.

Reply:
I'm thinking of the order of items in the drop-down list, but that's probably the other structure.

Resolving.